wkcuber/README.md

```diff
@@ -159,25 +53,6 @@ poetry run pylint -j4 wkcuber
 poetry run pytest tests
 ```
 
-Please, run the extended test suite:
-```bash
-tests/scripts/all_tests.sh
-```
-
-PyPi releases are automatically pushed when creating a new Git tag/Github release. 
-
-## API documentation
-Check out the [latest version of the API documentation](https://docs.webknossos.org/api/wkcuber.html).
```

Comment:
I guess we publish the API documentation at https://docs.webknossos.org/api/wkcuber.html

@normanrz I am not sure about the usefulness of the API documents and if we still need them as the wkcuber is now such a lightweight wrapper. 

@markbader If we indeed remove them, then please remove `docs/generate.sh` and  adjust the CI and general documentation generation workflows accordingly.

Reply:
The webknossos cli is now described in the webknossos-py docs and the CI is adapted accordingly.